wrappers/python/xraydb/xraydb.py

```diff
@@ -430,8 +430,29 @@ def xray_lines(self, element, initial_level=None, excitation_energy=None):
                                                r.initial_level, r.final_level)
         return out
 
+    def xray_line_strengths(self, element):
+        """
+        return the absolute line strength in cm^2/gr for all available lines
+
+        arguments
+        ---------
+        element:  atomic number, atomic symbol for element
+
+        """
+        out = {}
+        for label, eline in self.xray_lines(element).items():
```

Comment:
This is just returning line strength in cm^/gr, which does not depend on energy.   It would be reasonable to add incident energy (and perhaps minimum energy too) as filter on what lines get returned.